src/ui/public/chrome/index.d.ts

```diff
@@ -26,6 +26,7 @@ declare class Chrome {
   public dangerouslyGetActiveInjector(): Promise<IInjector>;
   public getBasePath(): string;
   public getXsrfToken(): string;
+  public getKibanaVersion(): string;
```

Comment:
did you drop a file in this PR?  I know you added this because a file you added typescript to needed it... but now I don't see it imported in this PR.

Reply:
Yep I didn't end up converting panel_state.js -> .ts at this point. Should I revert the import out of idnex.d.ts until that?

Reply:
Nope, that's fine, just was curious!